Enable k8s configmaps as flags for play kube (#7910)
containers/podman: docs/source/markdown/podman-play-kube.1.md

Use certificates at *path* (\*.crt, \*.cert, \*.key) to connect to the registry.
Default certificates directory is _/etc/containers/certs.d_. (Not available for remote commands)

**--configmap**=*path*

> Use kubernetes configmap yaml at path to provide a source for environment variable values within the containers of the pod.

Reviewer: I would prefer to stick to the uppercase spelling everywhere:
```suggestion
Use Kubernetes configmap yaml at path to provide a source for environment variable values within the containers of the pod.
```

> Note: You can use *--configmap* flag n number times or a comma-separated list of paths to pass multiple kubernetes configmap yamls.

Reviewer 2: We try to avoid the words "you" and "we" in man page docs.  Suggest:
```suggestion
Note: The *--configmap* option can be used multiple times or a comma-separated list of paths can be used to pass multiple kubernetes configmap yamls.
```

Reviewer 2: Please add an example or two for this option in the examples section.

Author: done. I added two examples as requested.
